panel_grid.py

```diff
@@ -14,8 +15,16 @@ def __init__(self, window, heigth, width):
         self.window = window
         self.heigth = heigth
         self.width  = width
-        self.activities = ['Activity 1', 'Activity 2', 'Activity 3', 'Activity 4', 'Activity 5']
-        self.days = ['Monday','Tuesday','Wednesday','Thursday','Friday','Saturday','Sunday']
+        self.week = 1
+        self.activities = [['Activity 1', 'Activity 2', 'Activity 3', 'Activity 4', 'Activity 5']]
+        self.days = []
+        self.t = Timeline()
+        self.t.add_week()
+        self.days.append([])
+        
+        for i in range(len(self.t.timeline['week'+str(self.week)])):
```

Comment:
line 25 to 27 will definitly have to move to the show_table() function that will get "a" week from the Timeline object